src/types/packet.ts

```diff
@@ -63,6 +68,8 @@ function packetContext(frame: EthernetFrame): PacketContext {
       return { type: "HTTP", layer: Layer.App };
     }
   }
+  if (frame.payload.type() === ARP_PROTOCOL_TYPE)
+    return { type: "ARP", layer: Layer.Link };
```

Comment:
I think this one's supposed to be

```suggestion
    return { type: "ARP", layer: Layer.Network };
```

Reply:
arp protocol belongs to the link layer

Reply:
Yeah, but it's a frame payload, so unless this is set to network, it won't show the information on that payload right now.

Reply:
Ok